.ci/env/tbb.bat

```diff
@@ -1,6 +1,6 @@
 @echo off
 rem ============================================================================
-rem Copyright 2020 Intel Corporation
+rem Copyright 2022 Intel Corporation
```

Comment:
2020 was correct since we are setting the date the file was added

Reply:
Fixed